core.py

```diff
@@ -116,8 +119,13 @@ def sender(conn: socket, home_dir: str) -> None:
             command = message.split(" ")
             if command[0] == ":UPLOAD:" and len(command) == 2:
                 filename = message.split()[1]
-                upload(conn, f"{home_dir}/{filename}")
+                upload(conn, f"{home_dir}\{filename}")
```

Comment:
I see you changed the forward slash ('/') to the backward slash ('/') to make it work on Windows. We could create a variable and assign it the appropriate slash based on whether the system the program is running on is Mac or Windows.